tools/NewModuleScript/src/main/kotlin/NewModuleScript.kt

```diff
@@ -0,0 +1,327 @@
+        FileUtils.write(file, "# $sampleName\n", StandardCharsets.UTF_8)
+    } catch (e: IOException) {
+        e.printStackTrace()
+        exitProgram(e)
+    }
+
+    //Update README.metadata.json
+    file = File("$samplesRepoPath/samples/$sampleWithHyphen/README.metadata.json")
+    try {
+        FileUtils.write(
+            file,
+            """
+                {
+                    "category": "$sampleCategory",
+                    "description": "TODO",
+                    "formal_name": "$sampleNameCamelCase",
+                    "ignore": false,
+                    "images": [
+                        "$sampleWithHyphen.png"
+                    ],
+                    "keywords": [ ],
+                    "language": "kotlin",
+                    "redirect_from": "",
+                    "relevant_apis": [ ],
+                    "snippets": [
+                        "src/main/java/com/esri/arcgismaps/sample/$sampleWithoutSpaces/${sampleNameCamelCase}ViewModel.kt",
+                        "src/main/java/com/esri/arcgismaps/sample/$sampleWithoutSpaces/${sampleNameCamelCase}Screen.kt",
+                        "src/main/java/com/esri/arcgismaps/sample/$sampleWithoutSpaces/MainActivity.kt"
+                    ],
+                    "title": "$sampleName"
+                }
+                
+                """.trimIndent(), StandardCharsets.UTF_8
+        )
+    } catch (e: IOException) {
+        e.printStackTrace()
+        exitProgram(e)
+    }
+
+    //Update build.gradle.kts
+    file = File("$samplesRepoPath/samples/$sampleWithHyphen/build.gradle.kts")
+    try {
+        var fileContent = FileUtils.readFileToString(file, StandardCharsets.UTF_8)
+        fileContent = fileContent.replace("sample.displaycomposablemapview", "sample.$sampleWithoutSpaces")
+        FileUtils.write(file, fileContent, StandardCharsets.UTF_8)
+    } catch (e: IOException) {
+        e.printStackTrace()
+        exitProgram(e)
+    }
+
+    //Update strings.xml
+    file = File("$samplesRepoPath/samples/$sampleWithHyphen/src/main/res/values/strings.xml")
+    try {
+        var fileContent = FileUtils.readFileToString(file, StandardCharsets.UTF_8)
+        fileContent = fileContent.replace(
+            "<string name=\"display_composable_map_view_app_name\">Display composable map view</string>",
+            "<string name=\"${sampleNameUnderscore}_app_name\">$sampleName</string>"
+        )
+        FileUtils.write(file, fileContent, StandardCharsets.UTF_8)
+    } catch (e: IOException) {
+        e.printStackTrace()
+        exitProgram(e)
+    }
+
+    //Update MainActivity.kt
+    file =
+        File("$samplesRepoPath/samples/$sampleWithHyphen/src/main/java/com/esri/arcgismaps/sample/$sampleWithoutSpaces/MainActivity.kt")
+    try {
+        var fileContent = FileUtils.readFileToString(file, StandardCharsets.UTF_8)
+        fileContent = fileContent.replace("Copyright 2023", "Copyright " + Calendar.getInstance()[Calendar.YEAR])
```

Comment:
```suggestion
        fileContent = fileContent.replace("Copyright 2024", "Copyright " + Calendar.getInstance()[Calendar.YEAR])
```

Reply:
The template uses `2023`, this line is to replace that content with the _"current"_ year. (2024/2025/...)